cmake/libxmp-sources.cmake.in

```diff
@@ -0,0 +1,5 @@
+set(LIBXMP_SRC_LIST@SRCS@)
+
+set(LIBXMP_SRC_LIST_PROWIZARD@PSRCS@)
+
+set(LIBXMP_SRC_LIST_DEPACKERS@DSRCS@)
```

Comment:
Please add a space between name and `@`  (same with
the lite version.)

You possibly did that to avoid a trailing space at the first line,
but that's perfectly tolerable.

Reply:
I would have something like next (note, I don't like trailing spaces at all, I always cut them off)

```
set(LIBXMP_SRC_LIST_PROWIZARD
@PSRCS@)

set(LIBXMP_SRC_LIST_DEPACKERS
@DSRCS@)
```

And the generator something like (approximately, may be wrong, please fix, also, use 4 spaces indent)
```make
cmake-prepare:
	@echo Generate cmake/libxmp-sources.cmake
	@sed -e 's!@SRCS@!    $(OBJS:.o=.c\n)\n!' \
	     -e 's!@PSRCS@!    $(PROWIZARD_OBJS:.o=.c\n)\n!' \
	     -e 's!@DSRCS@!    $(DEPACKER_OBJS:.o=.c\n)\n!' \
	     cmake/libxmp-sources.cmake.in > cmake/libxmp-sources.cmake
```

With a goal to get some:
```cmake
set(LIBXMP_SRC_LIST_DEPACKERS
    file1.c
    file2.c
    file3.c
    # ...
    filen.c
)
```